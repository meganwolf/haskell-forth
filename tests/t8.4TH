084 EMIT 104 EMIT 105 EMIT 115 EMIT 032 EMIT 115 EMIT 104 EMIT 111 EMIT 117 EMIT 108 EMIT 100 EMIT 032 EMIT 104 EMIT 097 EMIT 118 EMIT 101 EMIT 032 EMIT 115 EMIT 116 EMIT 117 EMIT 102 EMIT 102 EMIT 032 EMIT 108 EMIT 101 EMIT 102 EMIT 116 EMIT 032 EMIT 105 EMIT 110 EMIT 032 EMIT 116 EMIT 104 EMIT 101 EMIT 032 EMIT 115 EMIT 116 EMIT 097 EMIT 099 EMIT 107 EMIT 032 049 050 051 052 049 050 052 049 050 052 049 050 052