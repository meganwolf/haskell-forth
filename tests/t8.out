This should have stuff left in the stack
"Stack not empty"
[Integer 52,Integer 50,Integer 49,Integer 52,Integer 50,Integer 49,Integer 52,Integer 50,Integer 49,Integer 52,Integer 51,Integer 50,Integer 49,Integer 32]